server/main.lua

```diff
@@ -143,8 +109,6 @@ RegisterNetEvent('qb-cityhall:server:ApplyJob', function(job)
     TriggerClientEvent('QBCore:Notify', src, Lang:t('info.new_job', {job = JobInfo.label}), 'success')
 end)
 
-RegisterNetEvent('qb-cityhall:server:getIDs', giveStarterItems)
-
```

Comment:
We're not giving the id card and driver license as starting items anymore?

Reply:
It doesn't give them from here, but from multichar